generator/test_case_generator.go

```diff
@@ -42,6 +42,12 @@ var testTypeToTestDirMap = map[string][]string{
 		"./test/collection_interval",
 		"./test/metric_dimension",
 	},
+	/*
+		You can only place 1 mac instance on a dedicate host a single time.
+		Therefore, limit down the scope for testing in Mac since EC2 can be done with Linux
+		and Mac under the hood share similar plugins with Linux
+	*/
+	"ec2_mac": {},
```

Comment:
I think the generator will give an empty json array if nothing is here. Please confirm this works.

Reply:
Yes. As mentioned, I am not adding any test yet. Only adding the basic before onboarding the test to skimming down.